bbc/simorgh
# NVDA-FIX: Post h3 should be read with one key press

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-785

Overall changes
======
Fixes bug on NVDA so that Post heading timestamp and other content is read out with one key press. This was caused by the use of Flex and the <time> element.

Note this works on Chrome but not on Firefox due to another bug we've found (The position:absolute on the visually hidden element is causing text splitting)

Code changes
======
- Replaces flex with inline-block and lineheight override
- This should achieve the same UX, fix the NVDA bug and not cause regression on any other screen readers.

Testing
======
Test on a windows device:
1. Go to live page story book permalink (http://tinyurl.com/45pa8mt5) on Chrome (you can test on Firefox but this won't work as expected due to a different bug).
2. Open  NVDA
3. Navigate through Posts using the arrow key and ensure that 'timestamp, breaking, heading, subheading' are announced in one key press.

Required testing following code review
- UX review
- Test on screen readers to ensure no regression since A11y swarm

Helpful Links
======
[Figma Design](https://www.figma.com/file/kBQdRE6Tc5XZ9L2HejAUgD/Live-mvp---post-stream-%26-pagination---handover?type=design&node-id=1-2&mode=design&t=uB1JFXfyNxTfE2QW-0)

[Previous Swarm (Full Live page)](https://docs.google.com/spreadsheets/d/19hnmvykMfe3ycmNznFAUrbgH7LETGuP_2IpqolW608A/edit?usp=sharing)

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
